### PR TITLE
Issue #11214: Specify violation messages for EmptyForInitializerPadCheck Input Files

### PR DESCRIPTION
**Pull request by suniti0804**
Issue #11214

### CODE REVIEW

**Review by romani — APPROVED** [review]
Ok to merge

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad2.java [hunk 1]
```diff
@@ -7,5 +7,5 @@
 
 package com.puppycrawl.tools.checkstyle.checks.whitespace.emptyforinitializerpad;
 
-class InputEmptyForInitializerPad2 // violation
+class InputEmptyForInitializerPad2 // violation 'Invalid exception message'
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad2.java:10]: How did we get this message for this check?

https://checkstyle.org/checks/whitespace/emptyforinitializerpad.html#Violation_Messages
I am only seeing `is preceded with whitespace` or `is not preceded with whitespace`.

Comment (rnveach) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad2.java:10]: @suniti0804 Also, there is a conflict.

Comment (suniti0804) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad2.java]: > How did we get this message for this check?
> 
> https://checkstyle.org/checks/whitespace/emptyforinitializerpad.html#Violation_Messages I am only seeing `is preceded with whitespace` or `is not preceded with whitespace`.

actually, there is no any violation for this input class, so i removed the // violation comment, same has been found for an input file using RegexpCheckTest as well - https://github.com/checkstyle/checkstyle/pull/13965#discussion_r1375303142

Comment (suniti0804) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad2.java]: > How did we get this message for this check?
> 
> https://checkstyle.org/checks/whitespace/emptyforinitializerpad.html#Violation_Messages I am only seeing `is preceded with whitespace` or `is not preceded with whitespace`.

acually, in this input class there is no any violation of the kind specified, so in my latest commit i removed the // violation comment. same has been found for an input file using RegexpCheckTest - https://github.com/checkstyle/checkstyle/pull/13965#discussion_r1375303142

File: src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java [hunk 2]
```diff
@@ -141,8 +141,8 @@ public final class InlineConfigParser {
      */
     private static final Set<String> SUPPRESSED_CHECKS = new HashSet<>(Arrays.asList(
             "com.puppycrawl.tools.checkstyle.checks.regexp.RegexpCheck",
-            "com.puppycrawl.tools.checkstyle.checks.whitespace.EmptyForInitializerPadCheck",
-            "com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocStyleCheck"));
+            "com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocStyleCheck",
+            "com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck"));
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java:145]: Why are we adding `SummaryJavadocCheck`?

**Review by rnveach — CHANGES_REQUESTED** [review]

**Comment by suniti0804** [comment]
Rebased the branch as 1 check was not getting approved

File: src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java (hunk 2 above)
Comment (suniti0804) [reply src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java]: It appears there may have been a slight oversight on my part, as I was processing 2 checks concurrently. I have made the new changes and `SummaryJavadocCheck` has been removed in this PR - https://github.com/checkstyle/checkstyle/pull/13963

Comment (suniti0804) [reply src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java]: done

**Review by rnveach — APPROVED** [review]

**Pull request merged**